pypeapp/lib/log.py

```diff
@@ -44,10 +37,9 @@
 
 
 PYPE_DEBUG = int(os.getenv("PYPE_DEBUG", "0"))
-
-if _mongo_logging:
```

Comment:
This is required. Mongo logging is not required and when `pype install` is executed pymongo is not installed yet.

Reply:
Good point. Didnt test installation.